homeassistant/components/lyric/climate.py

```diff
@@ -207,12 +213,10 @@ def __init__(
             self._attr_hvac_modes.append(HVACMode.HEAT_COOL)
 
         # Setup supported features
-        if device.changeableValues.thermostatSetpointStatus:
+        if self._attr_thermostat_type == LyricThermostatType.LCC:
```

Comment:
I think it's best practice to use `is` when comparing to enums: https://docs.python.org/3/howto/enum.html#comparisons

Reply:
Thanks for the suggestion. I've made those changes.